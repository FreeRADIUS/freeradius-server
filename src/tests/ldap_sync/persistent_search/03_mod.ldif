#
# ARGV: -x -H ${PERSISTENT_SEARCH_TEST_SERVER} -D "cn=manager" -w "secret123"
#
dn: uid=fred,ou=people,dc=example,dc=com
changeType: modrdn
newrdn: uid=frederic
deleteoldrdn: 1

