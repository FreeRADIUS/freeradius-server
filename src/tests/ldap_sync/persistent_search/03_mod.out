Modify ou=people,dc=example,dc=com uid=frederic,ou=people,dc=example,dc=com uid=fred,ou=people,dc=example,dc=com frederic
